indexer/packages/postgres/src/db/migrations/migration_files/20240717160024_create_leaderboard_pnl_table.ts

```diff
@@ -16,7 +16,7 @@ export async function up(knex: Knex): Promise<void> {
     table.string('pnl').notNullable();
     table.string('currentEquity').notNullable();
     table.integer('rank').notNullable();
-    table.primary(['subaccountId', 'timeSpan']);
+    table.primary(['address', 'timeSpan']);
   });
```

Comment:
This is technically okay right now because no one has updated their indexer with this migration, but so you know this would not be possible if it was every deployed, and you would have to create another migration

Reply:
Yup, i agree with you here. would be terrible if this was deployed